ui/src/turing/components/form/standard_ensembler/LinkedExperimentsContextMenu.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import {
+  EuiContextMenu,
+  EuiPopover,
+  EuiButtonEmpty,
+  EuiTextColor,
+  EuiIcon
+} from "@elastic/eui";
+
+export const LinkedExperimentsContextMenu = ({
+  item,
+  projectId,
+  routeToExperimentMappings,
+  isButtonPopoverOpen,
+  setIsButtonPopoverOpen,
+  experimentStatus
+}) => {
+  let numRoutes = routeToExperimentMappings[item.id] ? Object.keys(routeToExperimentMappings[item.id][experimentStatus]).length : 0;
+
+  const onButtonClick = () => {
+    let newIsButtonPopoverOpen = { ...isButtonPopoverOpen };
+    newIsButtonPopoverOpen[item.id][experimentStatus] = !isButtonPopoverOpen[item.id][experimentStatus];
+    setIsButtonPopoverOpen(newIsButtonPopoverOpen);
+  };
+
+  const closePopover = () => {
+    let newIsButtonPopoverOpen = { ...isButtonPopoverOpen };
+    newIsButtonPopoverOpen[item.id][experimentStatus] = false;
+    setIsButtonPopoverOpen(newIsButtonPopoverOpen);
+  };
+
+  const button = (
+    <EuiButtonEmpty
+      size={"s"}
+      iconType={"arrowRight"}
+      iconSide={"right"}
+      color={"primary"}
+      onClick={onButtonClick}
+      isDisabled={numRoutes === 0}
+    >
+      {numRoutes}
+    </EuiButtonEmpty>
+  );
+
+  return isButtonPopoverOpen[item.id] ? (
+    <EuiPopover
+      button={button}
+      isOpen={isButtonPopoverOpen[item.id][experimentStatus]}
+      closePopover={closePopover}
+      panelPaddingSize={"none"}
+      anchorPosition={"rightCenter"}
+    >
+      <EuiContextMenu
+        initialPanelId={0}
+        panels={
+          [
+            {
+              id: 0,
+              title: `Linked ${experimentStatus[0].toUpperCase() + experimentStatus.slice(1)} Experiments`,
+              items: Object.values(routeToExperimentMappings[item.id][experimentStatus]).map(e => (
+                {
+                  name: (
+                    <EuiTextColor>
```

Comment:
Is `EuiTextColor` necessary here? Rather than using `html a` tag, can we use `EuiLink` instead?

Reply:
`EuiTextColor`: Oops not necessary, forgot to remove that. As for `EuiLink`, why would you prefer it to the `a` tag? I can use either one, no real preference except for some additional free styling we get.

Reply:
I think considering that we're using `eui` libraries and React, it's cleaner to use these components rather than html tags unless we need to do something specific/not achievable via these components.